packages/esm-patient-forms-app/src/forms/form-view.component.tsx

```diff
@@ -30,6 +30,7 @@ import { EditIcon, formatDatetime, useConfig, useLayoutType, usePagination } fro
 import { type ConfigObject } from '../config-schema';
 import { type CompletedFormInfo } from '../types';
 import styles from './form-view.scss';
+import { mapFormsToHtmlFormEntryForms } from '../form-entry-interop';
```

Comment:
Following our coding conventions (imports without an org or file path, imports with an org, imports with a file path, any stylesheets, all groups organized alphabetically):

```suggestion
import { type ConfigObject } from '../config-schema';
import { mapFormsToHtmlFormEntryForms } from '../form-entry-interop';
import { type CompletedFormInfo } from '../types';
import styles from './form-view.scss';
```